blocks/v2-text-with-images/v2-text-with-images.css

```diff
@@ -0,0 +1,158 @@
+.v2-text-with-images {
+  --text-with-images-image-gap: 12px;
+  --text-with-images-next-slide-preview: 22px;
+
+  padding: 24px 0;
+}
+
+.v2-text-with-images__row {
+  display: flex;
+  flex-direction: column;
+}
+
+.v2-text-with-images__col {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  color: var(--c-white);
+  padding-bottom: 40px;
+  max-width: 500px;
+  align-self: center;
+}
+
+.v2-text-with-images__pretitle {
+  font-feature-settings: 'clig' off, 'liga' off;
+  font-size: var(--f-subtitle-2-font-size);
+  letter-spacing: var(--f-subtitle-2-letter-spacing);
+  line-height: var(--f-subtitle-2-line-height);
+  color: var(--c-grey-3);
+}
+
+.v2-text-with-images__heading {
+  font-family: var(--ff-volvo-novum-medium);
+  font-size: var(--f-heading-4-font-size);
+  letter-spacing: var(--f-heading-4-letter-spacing);
+  line-height: var(--f-heading-4-line-height);
+  margin: 16px 0 8px;
+}
+
+.v2-text-with-images__text {
+  margin: 0 0 16px;
+}
+
+.v2-text-with-images__images-list-col {
+  max-width: unset;
+  padding: 0;
+  margin: 0 -16px;
+}
+
+.v2-text-with-images__images-list {
+  display: flex;
+  flex-flow: row nowrap;
+  list-style: none;
+  overflow: auto;
+  padding-left: 0;
+  scroll-snap-type: x mandatory;
+  margin: 0;
+  gap: 12px;
+}
+
+.v2-text-with-images__images-list::-webkit-scrollbar {
+  display: none;
+}
+
+.v2-text-with-images__images-list-item {
```

Comment:
Please adjust a max width in tablet version, because if the viewport is 1100px, the image is huge.

Reply:
max-widht added